index.html

```diff
@@ -14,8 +14,98 @@
 				<li><a href="#">Contact</a></li>
 			</ul>
 		</nav>
-		
-
 	</div>
+	<section class="F-section"><p class="T-name">Amin Dream Team</p>
```

Comment:
classnames are best done in lowercase letters as far as I know.

```suggestion
class="team-name"
```

team name is a more descriptive than T-name. Same with F-section. I have no idea what F stands for here.